nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/flow/change-version-dialog/change-version-dialog.html

```diff
@@ -22,32 +22,48 @@ <h2 mat-dialog-title>
     <mat-dialog-content>
         <div class="dialog-content flex flex-col gap-y-4 w-full">
             <div class="flex flex-row w-full">
-                <div class="flex flex-1 flex-col gap-y-4">
+                <div class="flex w-1/2 flex-col gap-y-4 overflow-hidden">
```

Comment:
Total nit pick here but we don't need overflow-hidden here: 

```suggestion
                <div class="flex w-1/2 flex-col gap-y-4">
```